Project2015To2017.Console/Program.cs

```diff
@@ -19,67 +13,9 @@ static void Main(string[] args)
 
 		private static void ConvertProject(Options options)
 		{
-			var conversionOptions = options.ConversionOptions;
-
-			var convertedProjects = new List<Project>();
-
 			ILogger logger = new ConsoleLogger("console", (s, l) => l >= LogLevel.Information, true);
 
-			logger.LogError("csproj-to-2017 is obsolete and will be removed very soon. Migrate to Project2015To2017.Migrate2017.Tool (dotnet migrate-2017) as soon as possible.");
-
-			foreach (var file in options.Files)
-			{
-				var projects = new ProjectConverter(logger, Vs15TransformationSet.Instance, conversionOptions)
-					.Convert(file, logger)
-					.Where(x => x != null)
-					.ToList();
-				convertedProjects.AddRange(projects);
-			}
-
-			System.Console.Out.Flush();
-
-			var analyzer = new Analyzer<LoggerReporter, LoggerReporterOptions>(new LoggerReporter(logger));
-			foreach (var project in convertedProjects)
-			{
-				analyzer.Analyze(project);
-			}
-
-			if (options.DryRun)
-			{
-				return;
-			}
-
-			var doBackup = !options.NoBackup;
-
-			var writer = new Writing.ProjectWriter(logger, x => x.Delete(), _ => { });
-			foreach (var project in convertedProjects)
-			{
-				writer.Write(project, doBackup);
-			}
-
-			System.Console.Out.Flush();
-
-			logger.LogInformation("### Performing 2nd pass to analyze converted projects...");
-
-			conversionOptions.ProjectCache?.Purge();
-
-			convertedProjects.Clear();
-
-			foreach (var file in options.Files)
-			{
-				var projects = new ProjectConverter(logger, BasicReadTransformationSet.Instance, conversionOptions)
-					.Convert(file, logger)
-					.Where(x => x != null)
-					.ToList();
-				convertedProjects.AddRange(projects);
-			}
-
-			System.Console.Out.Flush();
-
-			foreach (var project in convertedProjects)
-			{
-				analyzer.Analyze(project);
-			}
+			logger.LogError("csproj-to-2017 is obsolete. Use Project2015To2017.Migrate2017.Tool (dotnet migrate-2017).");
```

Comment:
Can we include a step by step guide for users? 
So dotnet install...
Dotnet migrate-2017 migrate bla.csproj

That way if they encounter this they don’t need to open a browser but can just run what it says there.